cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionCheckCommand.java

```diff
@@ -273,10 +273,14 @@ public class ExtractionCheckCommand extends Command {
       description = "Full git commit sha, used for setting a status on a commit in Github.")
   String commitSha;
 
-  @Autowired RestTemplate restTemplate;
+  RestTemplate restTemplate;
```

Comment:
It might be better to just do the creation in the reportStatistics method rather than leaving it as a class variable, to avoid confusion

Reply:
I cannot do that because we have a unit tests which mock out the member to test that there is an error reported if sending stats fails